services/api/src/admin_new_rsu.py

```diff
@@ -85,12 +93,34 @@ def check_safe_input(rsu_spec):
     return True
 
 
-def add_rsu(rsu_spec):
+def enforce_add_rsu_org_permissions(
+    *,
+    user: EnvironWithOrg,
+    rsu_spec: dict,
+):
+    if not user.user_info.super_user:
+        qualified_orgs = user.qualified_orgs
+        unqualified_orgs = [
+            org
+            for org in rsu_spec.get("organizations", [])
+            if org not in qualified_orgs
+        ]
+        if unqualified_orgs:
+            raise UnauthorizedException(
+                f"Unauthorized added organizations: {','.join(unqualified_orgs)}"
+            )
+
+
+@require_permission(
```

Comment:
This seems to be doubly checking the operator role in at least when POST is called. Is this just to enforce the additional check?